example/redis/distros/selector-repro.sh

```
@@ -0,0 +1,71 @@
+na=$(kubectl -n "$NA_NS" get pods -o wide --field-selector spec.nodeName="$master_node" --no-headers | awk '/node-agent/{print $1; exit}')
+logs=$(kubectl -n "$NA_NS" logs "$na" --since="${DURATION}s")
+
+ipof() { kubectl -n "$NS" get pod -l "repro=$1" -o jsonpath='{.items[0].status.podIP}'; }
+count() { printf '%s\n' "$logs" | grep -c "ingress network communication from: $1" || true; }
+
+printf '\n%-18s %-12s %-11s %-6s %-8s\n' client node allowlisted R0012 verdict
+for row in "sel-allow-intra|app.kubernetes.io/name=redis-client|$master_node|yes|0" \
+           "sel-rogue-intra|app=rogue|$master_node|no|>0" \
+           "sel-allow-inter|app.kubernetes.io/name=redis-client|$other_node|yes|0" \
+           "sel-rogue-inter|app=rogue|$other_node|no|>0"; do
+  IFS='|' read -r name sel node allow want <<<"$row"
```

Shellcheck (0.11.0)

[warning] sel appears unused. Verify use (or export if used externally).

(SC2034)